### DISCUSSION TITLE
Add support for shortcuts with fn and duplicated arrows

### DISCUSSION THREAD

**Discussion by LagerstedtMarcus** [rxhanson/Rectangle#1310] [Ideas]
I want to be able to construct my shortcuts as I want to, today it's limited to what is decided. 
1. I can't combine arrow keys like ⬆️ + ➡️  for the top right corner. 
2. Also to be able to use the fn key for shortcuts.

**Comment by rxhanson** [comment]
These 2 limitations are due to how macOS handles keyboard shortcuts. In order to add these in, the app would have to listen to all keystrokes, not just register keyboard shortcuts. Because of this, I haven't added in this functionality, but I wouldn't be opposed to a good pull request that adds these in.
